packages/backend/src/modules/bridges/plugins/mayan-swift.ts

```diff
@@ -38,10 +49,43 @@ export class MayanSwiftPlugin implements BridgePlugin {
       })
     }
 
+    const abi =
```

Comment:
The code that follows is absolutely horrible and I'm only approving it with the idea that it will be refactored